hubspot/management/commands/configure_hubspot_bridge.py

```diff
@@ -87,7 +87,19 @@
                 "fieldType": "text",
             },
         ],
-    }
+    },
+    "contacts": {
+        "properties": [
+            {
+                "name": "highest_education",
+                "label": "Highest Education",
+                "description": "Highest education level",
+                "groupName": "contactinformation",
+                "type": "string",
```

Comment:
Whether "string" or "number" should be used here will depend on what is saved in the database (an integer or string) and appears in the serializer.

Reply:
@pdpinch Could you please answer the question above saving integer or string as choices.